src/main/java/org/folio/service/manager/input/InputDataManagerImpl.java

```diff
@@ -249,12 +249,13 @@ private ExportPayload createExportPayload(OkapiConnectionParams okapiParams, Fil
     return exportPayload;
   }
 
-  private FileDefinition createExportFileDefinition(FileDefinition requestFileDefinition) {
-    String fileNameWithoutExtension = FilenameUtils.getBaseName(requestFileDefinition.getFileName());
+  private FileDefinition createExportFileDefinition(ExportRequest exportRequest) {
+    String fileNameWithoutExtension = FilenameUtils.getBaseName(exportRequest.getFileDefinition().getFileName());
     return new FileDefinition()
       .withFileName(fileNameWithoutExtension + DELIMITER + getCurrentTimestamp() + MARC_FILE_EXTENSION)
       .withStatus(FileDefinition.Status.IN_PROGRESS)
-      .withJobExecutionId(requestFileDefinition.getJobExecutionId());
+      .withJobExecutionId(exportRequest.getFileDefinition().getJobExecutionId())
+      .withMetadata(exportRequest.getMetadata());
```

Comment:
@andrii-novytskyi  You may have to factor this into your PR as well :) Thanks!

Reply:
@KVupp i will merge master into my branch after bugfix release